#
# patch created with `svn diff -r61 . > patches/regex_vlan.patch`
# on Tue Jul 10 21:29:49 CEST 2007
# applied on r61
# 
# bug originally reported on ubuntu:
# https://bugs.edge.launchpad.net/ubuntu/+source/ddclient/+bug/77570
# posted upstream as bug #1747337
# patch written by marclogghe
#
Index: ddclient
===================================================================
--- ddclient	(revision 61)
+++ ddclient	(working copy)
@@ -1601,7 +1601,7 @@
 	return undef if $value eq "";
 
     } elsif ($type eq T_IF) {
-	return undef if $value !~ /^[a-z0-9:_-]+$/;
+	return undef if $value !~ /^[a-z0-9:._-]+$/;
 
     } elsif ($type eq T_PROG) {
 	return undef if $value eq "";
